test/python/test_docker_pack.py

```diff
@@ -0,0 +1,288 @@
+#!/usr/bin/python3
+
+import pytest
+import os
+import docker
+import re
+import requests
+import time
+import subprocess
+import tarfile
+
+from utils import project_name
+from utils import basepath
+from utils import tarantool_version
+from utils import tarantool_enterprise_is_used
+from utils import recursive_listdir
+from utils import assert_dir_contents
+from utils import assert_filemodes
+
+
+# #######
+# Helpers
+# #######
+def find_image(docker_client, project_name):
+    for image in docker_client.images.list():
+        for t in image.tags:
+            if t.startswith(project_name):
+                return t
+
+
+def run_command_on_image(docker_client, image_name, command):
+    command = '/bin/bash -c "{}"'.format(command.replace('"', '\\"'))
+    output = docker_client.containers.run(
+        image_name,
+        command,
+        remove=True
+    )
+    return output.decode("utf-8").strip()
+
+
+def wait_for_container_start(container, timeout=10):
+    time_start = time.time()
+    while True:
+        now = time.time()
+        if now > time_start + timeout:
+            break
+
+        container_logs = container.logs(since=int(time_start)).decode('utf-8')
+        if 'entering the event loop' in container_logs:
+            return True
+
+        time.sleep(1)
+
+    return False
+
+
+# ########
+# Fixtures
+# ########
+@pytest.fixture(scope="module")
+def docker_client():
+    client = docker.from_env()
```

Comment:
Should we disable docker tests or show beauty message in case then docker is unavailable

Reply:
I'm not sure.
I think it's important to test docker so I don't like the idea of disabling this tests.
And I sure that python will fail with some beauty message anyway.